### PR TITLE
docs(Tree): update docs

### PR DESCRIPTION
**Pull request by himushroom**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [X] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Workflow
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |   Update docs about tree       |
| 🇨🇳 Chinese |   更新树组件相关文档        |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [X] Doc is updated/provided or not needed
- [X] Demo is updated/provided or not needed
- [X] TypeScript definition is updated/provided or not needed
- [X] Changelog is provided or not needed

---

### <samp>🤖 Generated by Copilot at d9c0072</samp>

Updated the Chinese documentation of the `Tree` component to explain how to enable multiple selection with the `selectedKeys` prop.

### <samp>🤖 Generated by Copilot at d9c0072</samp>

* Clarify the usage of `selectedKeys` prop for multiple selection in tree component ([link](https://github.com/ant-design/ant-design/pull/42989/files?diff=unified&w=0#diff-0bdf37a1fc0ec124fb3aa391932eaa134bd1de8c6f766f5e4082edbc30ce6c42L66-R66))

### CODE REVIEW

File: components/tree/index.zh-CN.md [hunk 1]
```diff
@@ -63,7 +63,7 @@ demo:
 | rootClassName | 添加在 Tree 最外层的 className | string | - | 4.20.0 |
 | rootStyle | 添加在 Tree 最外层的 style | CSSProperties | - | 4.20.0 |
 | selectable | 是否可选中 | boolean | true |  |
-| selectedKeys | （受控）设置选中的树节点 | string\[] | - |  |
+| selectedKeys | （受控）设置选中的树节点，多选需设置 `multiple` 为 true | string\[] | - |  |
```
Comment (li-jia-nan) [inline components/tree/index.zh-CN.md:66]: 英文文档也顺便补充一下？

Comment (afc163) [inline components/tree/index.zh-CN.md:66]:
```suggestion
| selectedKeys | （受控）设置选中的树节点，`multiple` 为 `true` 时生效。 | string\[] | - |  |
```

Comment (himushroom) [reply components/tree/index.zh-CN.md:66]: > 

改为生效是否有歧义？因为不加multiple时selectedKeys也是生效的，只是只能选中一个

Comment (afc163) [reply components/tree/index.zh-CN.md:66]: 也有道理。

**Review by afc163 — APPROVED** [review]

**Pull request merged**